node/full_node_integration_test.go

```diff
@@ -147,7 +150,7 @@ func TestLazyAggregator(t *testing.T) {
 	key, _, _ := crypto.GenerateEd25519Key(rand.Reader)
 	genesisValidators, signingKey := getGenesisValidatorSetWithSigner(1)
 	blockManagerConfig := config.BlockManagerConfig{
-		BlockTime:   1 * time.Second,
+		BlockTime:   1 * time.Millisecond,
```

Comment:
Let's make a follow up issue to make a constant for this so we are consistent across tests. This will help with flakiness since we can fine tune to a standard blocktime.